x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -271,6 +271,11 @@ export interface ActionDetails<TOutputContent extends object = object> {
    * This is an Array because the action could have been sent to multiple endpoints.
    */
   agents: string[];
+  /**
+   * The endpoint ID/fleet agent ID which was used to create the action (same as above) and the agent's name
+   * Action could have been sent for multiple endpoints which is why this is an Array
+   */
```

Comment:
We changed this to an abject now, so maybe:

```suggestion
/**
   * A map of `Agent ID`'s to which the action was sent whose value contains more information about the host (currently the host name only). NOTE: this property is only present if the `showHostInfo` API param is used.
   */
```

Reply:
2873f182e103dfede7e0e1d6c56e10f916a5f281 😅